.github/ISSUE_TEMPLATE/3-issue-existing-plugin.yml

```
@@ -0,0 +1,36 @@
+name: "🗃️ Functionality around a specific plugin"
+description: I have an issue or feature request around a specific, existing plugin.
+body:
+  - type: input
+    id: plugin
+    attributes:
+      label: Platform or tool
+      description: Which plugin does this concern?
+      placeholder: e.g. AWS, Stripe, GitHub
+    validations:
+      required: true
+  - type: textarea
+    id: desired
+    attributes:
+      label: Desired or expected behavior
+      description: What would you like to see this plugin do that it currently doesn't do (well)?
```

What about the following to encourage sharing the problem the user is facing:


```suggestion
      description: What do you want to do with this plugin that is currently not possible or does not work well?
```